packages/backend/src/postHog.ts

```diff
@@ -86,6 +99,7 @@ export async function isFeatureFlagEnabled(
                       groups: {
                           organization: user.organizationUuid,
                       },
+                      ...groupProperties,
```

Comment:
shouldn't it be:


```suggestion
                      groupProperties,
```

instead?